.github/main.workflow

```diff
@@ -0,0 +1,8 @@
+workflow "Code linting" {
+  on = "push"
+  resolves = ["GitHub Action for Flake8"]
+}
+
+action "GitHub Action for Flake8" {
+  uses = "cclauss/GitHub-Action-for-Flake8@0.0.1"
```

Comment:
Isn't it required to pass `args` parameter to the workflow file?

Reply:
Depending on whether you want to override the default one. But probably you're right..

Reply:
Done